index.js

```diff
@@ -1,3 +1,19 @@
-// javascript
+// Capture user's input on form submission
+let form = document.querySelector("form");
 
-alert("Hello from index.js!");
+form.addEventListener("submit", function (event) {
+  event.preventDefault();
+
+  console.log("Form submitted");
+});
```

Comment:
These code changes look good overall, well done:

1. **Event Handling**:
   - You added an event listener for form submission which prevents the default action and logs "Form submitted" to the console. This is a great start for handling form submissions.

2. **Logging**:
   - The selected line `console.log("Form submitted");` is a simple log statement to confirm that the form has been submitted. This is useful for debugging, and we will replace this with actual form handling logic eventually.

3. **Comments**:
   - You have added a block of comments describing the steps to handle and validate the form input. This is good for planning, and now these comments will be accompanied by actual implementation.

Overall, you've made a good start by setting up form submission handling and planning the necessary steps. The next step is to implement the planned functionality.